server/lib/txgh-server/webhooks/gitlab/blank_attributes.rb

```diff
@@ -0,0 +1,8 @@
+module TxghServer
+  module Webhooks
+    module Gitlab
+      class BlankAttributes < TxghServer::Webhooks::Github::BlankAttributes
```

Comment:
The inheritance here is fine, but it's a little weird at the same time. What would you think about introducing a base class like `Git::BlankAttributes` and creating two derived classes, one for github and one for gitlab? The two derived classes may not even be necessary if the base class contains all the needed functionality.